components/builder-originsrv/src/migrations/origin_public_keys.rs

```diff
@@ -83,11 +83,24 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
-
-    migrator.migrate("originsrv",
-                     r#"ALTER TABLE origin_public_keys
+    migrator
+        .migrate("originsrv",
+                 r#"ALTER TABLE origin_public_keys
                         DROP CONSTRAINT IF EXISTS
                           origin_public_keys_full_name_key"#)?;
-
+    migrator
+        .migrate("originsrv-2",
```

Comment:
May be a good idea to wrap both in a transaction to prevent dupes from sneaking in between the two running.